src/SearchInput/SearchInput.js

```diff
@@ -15,12 +15,16 @@ class SearchInput extends Component {
         this.state = {
             isExpanded: false,
             searchExpanded: false,
-            value: '',
-            searchList: this.props.searchList,
-            filteredResult: this.props.searchList
+            value: props.inputProps && props.inputProps.value ? props.inputProps.value : '',
```

Comment:
We can simplify this to `props.inputProps?.value ? props.inputProps.value : ''` - same with line 20 :)

Reply:
Updated.Thanks.